modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -49,6 +59,67 @@
 				title: editor.lang.link.title,
 			});
 
+			editor.ui.addBalloonToolbarButton('Twitter', {
+				_MAX_TWEET_LENGTH: 280,
+				_getHref(url, via) {
+					const selectedText = editor
+						.getSelection()
+						.getSelectedText()
+						.substring(0, this._MAX_TWEET_LENGTH);
+
+					// TODO - These parameters should be revisited when to use `url` and `via`.
```

Comment:
but when trying to tweet a text everything works. In AlloyEditor demo, these props are undefined